init.sh

```diff
@@ -36,6 +33,8 @@ else
 if [ -n "$(grep $IPName $ETC_HOSTS)" ]; then
 	echo "$IPName already exists : $(grep $IPName $ETC_HOSTS)"
 else
+    PublicIP="$(ping -c 1 $IPLoadBalancer | grep -oE '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}')"
```

Comment:
This will include the same IP address twice, one with the defined hostname and one blank. Is it possible to handle this?

Reply:
It doesn't output 2 ips for AWS ELBs :)